mypy_django_plugin/transformers/managers.py

```diff
@@ -139,6 +143,15 @@ def create_new_manager_class_from_from_queryset_method(ctx: DynamicClassDefConte
     """
     semanal_api = helpers.get_semanal_api(ctx)
 
+    # Do not interfere with `fail_if_manager_type_created_in_model_body`:
+    # stop adding new manager, if we are inside, and fall back to any
+    if semanal_api.scope.classes:
```

Comment:
Not sure if/what is more or less preferable

```suggestion
    if semanal_api.is_class_scope():
```

Reply:
Not the best idea: `scope.classes` will match function scope too, while `is_class_scope` checks [exactly](https://github.com/python/mypy/blob/master/mypy/semanal.py#L5164=), so inside function we'll be confused. I doubt it is important and has a use case, but logically `scope.classes` is preferable.

Reply:
Alright, yeah I bet allowing function scope might perhaps create some other troubles anyways. So doing `is_class_scope` would still be supporting functions halfly, probably.

Lets stick with `scope.classes`